internal/eval/eval.go

```diff
@@ -4108,6 +4110,91 @@ func evalPFMERGE(args []string, store *dstore.Store) []byte {
 	return clientio.RespOK
 }
 
+func evalPFDEBUG(args []string, store *dstore.Store) []byte {
+	if len(args) != 2 {
+		return diceerrors.NewErrArity("PFDEBUG")
+	}
+
+	subcommand := strings.ToUpper(args[0])
+	key := args[1]
+
+	obj := store.Get(key)
+	if obj == nil {
+		return diceerrors.NewErrWithMessage("Key does not exist")
+	}
+
+	hll, ok := obj.Value.(*hyperloglog.Sketch)
+	if !ok {
+		return diceerrors.NewErrWithMessage(diceerrors.WrongTypeHllErr)
+	}
+
+	sparseField := reflect.ValueOf(hll).Elem().FieldByName("sparseList")
+	sparseFieldValue := reflect.NewAt(sparseField.Type(), unsafe.Pointer(sparseField.UnsafeAddr())).Elem()
+
+	var encoding string
+	if sparseFieldValue.IsNil() {
+		encoding = "dense"
+	} else {
+		encoding = "sparse"
+	}
+
+	switch subcommand {
+	case "GETREG":
+		// For call to GETREG, encoding is converted from sparse to dense.
+		// We'll merge a sparse and an empty dense HLL to get a dense HLL
+		// with values of sparse HLL.
+		if encoding == "sparse" {
```

Comment:
Define this string in a constant. It would also be good to be consistent with the casing of these constants.